src/GitHubCreateMergePRs/config.xml

```diff
@@ -8,19 +8,22 @@
     <merge from="release/dev16.3" to="release/dev16.4" />
     <merge from="release/dev16.4" to="release/dev16.5" />
     <merge from="release/dev16.5" to="release/dev16.6" />
-    <merge from="release/dev16.6" to="master" />
+    <merge from="release/dev16.6" to="release/dev17.0-preview1" />
```

Comment:
should be release/dev16.7-preview1 right?  same for all below

Reply:
isn't master 17.0-preview1? That's what we're branching off

Reply:
https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/8755/Dev16.7-Schedule

Reply:
master is dev16.7-preview1, which will change into dev16.7-preview2 after the snap, I think dev17 is a ways off yet 😆

Reply:
duh I can't read

Reply:
updated